### DISCUSSION TITLE
Page is blocked from indexing, according to web.dev/meaure

### DISCUSSION THREAD

**Discussion by patarapolw** [vercel/next.js#14319] [Help] [answered]
Blocking Directive Source
--
X-Robots-Tag: noindex

![Screen Shot 2563-06-18 at 19 58 28](https://user-images.githubusercontent.com/21255931/85022877-172d1100-b19e-11ea-8b2b-3fcb368759b9.png)

This is the url -- https://jpdiary-git-next-privacy.patarapolw.vercel.app/

**Comment by timneutkens** [comment, marked as answer]
Staging urls are automatically set to noindex by Vercel to avoid accidentally indexing feature branches. Once you merge to the main branch and a production domain is aliased it won't have the noindex header.

**Reply by patarapolw** [reply]
I don't necessarily use "master" branch for production, though.

**Reply by timneutkens** [reply]
Doesn't really matter what the branch is named, as long as you add a production domain for the branch

**Reply by Izaark** [reply]
@timneutkens thanks !

**Reply by timB73** [reply]
@timneutkens I have a production domain and it still has noindex in the view source. I checked the code and there are no references to noindex :/

**Comment by ebrimaconta** [comment]
@timneutkens  I am having the same problem what should i do  i am using nextjs on ionos sever?
